virtio-devices/src/net.rs

```diff
@@ -449,7 +449,10 @@ impl Net {
     ) -> Result<Self> {
         assert!(!taps.is_empty());
 
+        #[cfg(not(fuzzing))]
         let mtu = taps[0].mtu().map_err(Error::TapError)? as u16;
+        #[cfg(fuzzing)]
+        let mtu = MIN_MTU;
```

Comment:
I'd prefer if you could create a `fuzzing` version of the `mtu()` function in `tap.rs`, that would return the MTU value you expect. This way you wouldn't have to modify the content of `new_with_tap()`.

Reply:
Good input. PR updated.